zebra-instructions.md

```diff
@@ -91,72 +97,151 @@ Install dependencies:
     sudo apt-get install redis
     redis-server
     ```
+
 2. Install and activate a node version manager (e.g. [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating))
 3. Install `boost` and `libsodium` development libraries
-   On Ubuntu:
+
     ```sh
     sudo apt install libboost-all-dev
     sudo apt install libsodium-dev
     ```
 
-Install `s-nomp`:
+#### Install `s-nomp`
+
 1. `git clone https://github.com/ZcashFoundation/s-nomp`
 2. `cd s-nomp`
 3. Use the Zebra fixes: `git checkout zebra-mining`
-4. Use node 8.17.0:
+4. Use node 8.11.0:
+
     ```sh
-    nodenv install 8.17.0
-    nodenv local 8.17.0
+    nodenv install 8.11.0
+    nodenv local 8.11.0
     ```
-    or 
+
+    or
+
     ```sh
-    nvm install 8.17.0
-    nvm use 8.17.0
+    nvm install 8.11.0
+    nvm use 8.11.0
     ```
 
 5. Update dependencies and install:
+
     ```sh
     export CXXFLAGS="-std=gnu++17"
     npm update
     npm install
     ```
 
-Run `s-nomp`:
+</details>
+
+<details><summary>Arch-specific instructions</summary>
+    
+#### Install dependencies
+
+1. Install [`redis`](https://redis.io/docs/getting-started/) and run it on the default port:
+
+    ```sh
+    sudo pacman -S redis
+    sudo systemctl start redis
+    ```
+
+2. Install and activate [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating):
+
+    ```sh
+    sudo pacman -S nvm
+    unset npm_config_prefix
+    source /usr/share/nvm/init-nvm.sh
+    ```
+
+3. Install `boost` and `libsodium` development libraries:
+
+    ```sh
+    sudo pacman -S boost libsodium
+    ```
+
+#### Install `s-nomp`
+
+1. `git clone https://github.com/ZcashFoundation/s-nomp && cd s-nomp`
+
+2. Use the Zebra configs: `git checkout zebra-mining`
+
+3. Use node 8.11.0:
+
+    ```sh
+    nvm install 8.11.0
+    nvm use 8.11.0
+    ```
+
+4. Update dependencies and install:
+
+    ```sh
+    npm update
+    npm install
+    ```
+
+</details>
+
+## Run `s-nomp`
+
 1. Edit `pool_configs/zcash.json` so `daemons[0].port` is your Zebra port
 2. Run `s-nomp` using `npm start`
 
 Note: the website will log an RPC error even when it is disabled in the config. This seems like a `s-nomp` bug.
 
 ## Install a CPU or GPU miner
 
-Install dependencies:
-1. Install a statically compiled `boost` and `icu`
-2. Optional: install static CUDA GPU mining libraries: https://github.com/nicehash/nheqminer#linux
+#### Install dependencies
+
+<details><summary>General instructions</summary>
+
+1. Install a statically compiled `boost` and `icu`.
+2. Install `cmake`.
+
+</details>
+
+<details><summary>Arch-specific instructions</summary>
+
+```sh
+sudo pacman -S cmake
```

Comment:
The Arch dependencies seem to be missing:
> Install a statically compiled `boost` and `icu`.

Reply:
Good catch, `boost` is installed in the previuous steps, so I omitted it. There's no harm in including it here again though.

Reply:
Done in [42d5a21](https://github.com/ZcashFoundation/s-nomp/pull/9/commits/42d5a21a05fa6c337b201f48ff90b884e40d7b90).